src/ostorlab/runtimes/definitions.py

```diff
@@ -265,6 +274,7 @@ def from_bus_message(cls, request):
             agent_def = AgentSettings(
                 key=agent.key,
                 version=agent.version,
+                custom_image=agent.custom_image if hasattr(agent, 'custom_image') else None,
```

Comment:
hasattr is not needed, commonly generated by AI Coder since they don't have the model definition in their context.